libs/auth/src/angular/login/login.component.html

```diff
@@ -37,11 +47,30 @@
     </bit-form-control>
 
     <div class="tw-grid tw-gap-3">
-      <!-- Continue button -->
-      <button type="button" bitButton block buttonType="primary" (click)="continuePressed()">
+      <!-- Continue button (sso is disabled) -->
+      <button
+        type="button"
+        bitButton
+        block
+        buttonType="primary"
+        class="vw-continue-login"
+        (click)="continuePressed()"
+      >
         {{ "continue" | i18n }}
       </button>
 
+      <!-- SSO button -->
+      <button
+        type="button"
+        bitButton
+        block
+        buttonType="primary"
+        class="vw-sso-login"
+        (click)="handleSsoClick()"
+      >
+        {{ "useSingleSignOn" | i18n }}
+      </button>
```

Comment:
I just noticed that you are missing `<i class="bwi bwi-provider tw-mr-1"></i>` which would be easier to spot if this button was not moved. So I think this should be left below the `passkey` login (without any changes) so future changes to the buttons are easier to spot.

Reply:
I missed it too, I'll check what it changes.
The position change is needed to be before the `or`.

Reply:
I think this is just the icon before the text:
<img width="477" height="50" alt="image" src="https://github.com/user-attachments/assets/eb1aed60-fd21-43b4-b6c8-7292bb8d3dcb" />